layouts/page/virtual-backgrounds.html

```diff
@@ -0,0 +1,60 @@
+{{ define "hero" }}
+    <header>
+        <div class="container mx-auto p-4 text-center flex-col m-24 mb-12">
+            <h1 class="text-5xl">{{ .Params.hero.title }}</h1>
+            <p class="mt-10 text-2xl">{{ .Params.hero.description }}</p>
+        </div>
+    </header>
+{{ end }}
+
+{{ define "main" }}
+    <section id="backgrounds" class="container mx-auto">
+        {{ range .Params.backgrounds }}
+            <div class="flex flex-col lg:flex-row my-4 lg:my-12">
+                <div class="w-full mb-12 order-last lg:order-first lg:mb-0 lg:w-5/12 p-4 px-16">
```

Comment:
```suggestion
                <div class="w-full mb-12 order-last lg:order-first lg:mb-0 lg:w-5/12 p-4 lg:px-16">
```

... otherwise you end up with more padding than you probably want on mobile.

Reply:
👍